crates/bevy_text/src/lib.rs

```diff
@@ -1,11 +1,40 @@
+//! This crate provides the tools for positioning and rendering text in Bevy.
+//!
+//! # `Font`
+//!
+//! Fonts contain information for drawing glyphs, which are shapes that typically represent a single character,
+//! but in some cases part of a "character" (grapheme clusters) or more than one character (ligatures).
+//!
+//! A font *face* is part of a font family,
+//! and is distinguished by its style (e.g. italic), its weight (e.g. bold) and its stretch (e.g. condensed).
+//!
+//! In Bevy, [`Font`]s are loaded by the [`FontLoader`] as assets,
+//!
+//! # `TextPipeline`
+//!
+//! The [`TextPipeline`] resource does all of the heavy lifting for rendering text.
+//!
+//! [`Text`] is first measured by creating a [`TextMeasureInfo`] in [`TextPipeline::create_text_measure`],
+//! which is called by a system.
+//!
+//! Note that text measurement is only relevant in a UI context.
```

Comment:
This comment isn't very clear to me, and I'm not sure why it might be helpful.

Reply:
It helps when working with this code I guess, since measuring the textsizes will have bearing on how large UI elements will be? 

Would like to leave this in tbh